docs/reference-guides/block-api/block-bindings.md

```diff
@@ -31,6 +31,7 @@ Right now, not all block attributes are compatible with block bindings. There is
 | Heading             | content                    |
 | Image               | id, url, title, alt        |
 | Button              | text, url, linkTarget, rel |
+| Navigation Link     | url                        |
```

Comment:
https://github.com/WordPress/gutenberg/pull/71994 is going to allow us to use the same functionality on Submenu block. We need to land that one first